src/Behat/Transliterator/SyncTool.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+namespace Behat\Transliterator;
+
+use RollingCurl\Request;
+use RollingCurl\RollingCurl;
+use Yaoi\Command;
+use Yaoi\Command\Option;
+use Yaoi\Http\Client;
+use Yaoi\String\Lexer\Parsed;
+use Yaoi\String\Lexer\Parser;
+use Yaoi\String\Lexer\Renderer;
+use Yaoi\String\Lexer\Token;
+use Yaoi\String\StringValue;
+use Yaoi\String\Parser as StringParser;
+
+/**
+ * Tool for converting char tables for Behat/Transliterator from Perl to PHP
+ *
+ * @package Behat\Transliterator
```

Comment:
we don't use the ``@package`` annotation. It serves no purpose when it duplicates the namespace